pages/badges.jsx

```diff
@@ -0,0 +1,192 @@
+                title: 'Problem Solving',
+                status: 'achieved',
+                description: '',
+                icon: '/img/components/badge-icon/ProblemSolving.png',
+                icon2x: '/img/components/badge-icon/ProblemSolving@2x.png'
+            },
+            {
+                key: 2,
+                title: 'Collaboration',
+                status: 'pending',
+                description: 'Lorem ipsum dolor sit amet',
+                icon: '/img/components/badge-icon/Collaborative.png',
+                icon2x: '/img/components/badge-icon/Collaborative@2x.png'
+            },
+            {
+                key: 3,
+                title: 'Creativity',
+                status: 'eligible',
+                description: 'Lorem ipsum dolor sit amet',
+                icon: '/img/components/badge-icon/Creativity.png',
+                icon2x: '/img/components/badge-icon/Creativity@2x.png'
+            },
+            {
+                key: 4,
+                title: 'Creativity',
+                status: 'eligible',
+                description: 'Lorem ipsum dolor sit amet',
+                icon: '/img/components/badge-icon/Creativity.png',
+                icon2x: '/img/components/badge-icon/Creativity@2x.png'
+            },
+            {
+                key: 5,
+                title: 'Problem Solving',
+                status: 'achieved',
+                description: '',
+                icon: '/img/components/badge-icon/ProblemSolving.png',
+                icon2x: '/img/components/badge-icon/ProblemSolving@2x.png'
+            },
+            {
+                key: 6,
+                title: 'Creativity',
+                status: 'eligible',
+                description: 'Lorem ipsum dolor sit amet',
+                icon: '/img/components/badge-icon/Creativity.png',
+                icon2x: '/img/components/badge-icon/Creativity@2x.png'
+            },
+            {
+                key: 7,
+                title: 'Creativity',
+                status: 'eligible',
+                description: 'Lorem ipsum dolor sit amet',
+                icon: '/img/components/badge-icon/Creativity.png',
+                icon2x: '/img/components/badge-icon/Creativity@2x.png'
+            },
+            {
+                key: 8,
+                title: 'Problem Solving',
+                status: 'achieved',
+                description: '',
+                icon: '/img/components/badge-icon/ProblemSolving.png',
+                icon2x: '/img/components/badge-icon/ProblemSolving@2x.png'
+            }
+        ];
+
+        return {
+            badges: dummyBadges
+        };
+    },
+    componentDidMount: function () {},
+    setBadgesData: function (error, data) {
+        var parsed = this.parseBadges(data);
+        this.setState({
+            badges: parsed
+        });
+    },
+    parseBadges: function (credlyResponse) {
+        // do parsing here
+        var data = credlyResponse;
```

Comment:
better to say `var data = []` here, and then use `credlyResponse` in the conditional, so we don't need an `else` later: